authorization/role/repository/role_mapping.go

```diff
@@ -77,26 +78,9 @@ func (m *GormRoleMappingRepository) TableName() string {
 }
 
 // CheckExists returns nil if the given ID exists otherwise returns an error
-func (m *GormRoleMappingRepository) CheckExists(ctx context.Context, ID uuid.UUID) (bool, error) {
+func (m *GormRoleMappingRepository) CheckExists(ctx context.Context, id string) error {
```

Comment:
Why has this been changed to `string` when the `role_mapping_id` column is a UUID?

Reply:
Just because base.CheckExists() uses string. We can keep UUID here though.

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/498/commits/a82bf3fd432bb4f3f5f7307f60d073c1a1cc3716